my-app/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
I noticed that you are using proptypes, but in order for them to work you need to have them in your dependency list! You can install proptypes by running `yarn add prop-types` in the my-app folder